### PR TITLE
deposit form: add responsive layout

### PR DESCRIPTION
**Pull request by jennur**
Closes [#445](https://github.com/inveniosoftware/react-invenio-deposit/issues/445)
- Stacked sidebar for tablet and mobile.

## Screenshots

__Tablet__
127 0 0 1_5000_uploads_new (3)

__Mobile__

### CODE REVIEW

**Review by kpsherva — APPROVED** [review]

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js [hunk 1]
```diff
@@ -154,246 +154,251 @@ export class RDMDepositForm extends Component {
...
+                      text: item.title_l10n,
+                      value: item.id,
+                      key: item.id,
+                    }))
+                  }
+                />
+                <DatesField options={this.vocabularies.metadata.dates} />
+                <VersionField />
+                <PublisherField />
+                <br />
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js:317]:
```suggestion
                <Divider />
```

File: invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js [hunk 2]
```diff
@@ -154,246 +154,251 @@ export class RDMDepositForm extends Component {
         <Container style={{ marginTop: "10px" }}>
```
Comment (kpsherva) [inline invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/deposit/RDMDepositForm.js:154]:
```suggestion
        <Container className="mt-10">
```

**Comment by kpsherva** [comment]
the PR needs rebase

**Comment by jennur** [comment]
# Updated screenshots

Note: The deposit form components also need responsive improvements. This PR is only for the overall layout.
Screenshot 2022-04-12 at 15 58 00
Screenshot 2022-04-12 at 15 58 21

**Review by ntarocco — APPROVED** [review]

**Pull request merged**